File: src/main/java/me/nepnep/nepbot/message/command/commands/animals/XmasRat.java
package me.nepnep.nepbot.message.command.commands.animals;

import me.nepnep.nepbot.message.command.ICommand;
import me.nepnep.nepbot.message.command.Category;
import net.dv8tion.jda.api.Permission;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.util.List;

public class XmasRat implements ICommand {
    @Override
    public void execute(List<String> args, GuildMessageReceivedEvent event) {
        event.getChannel().sendMessage("https://christmas.bigrat.monster/").queue();
    }
    @Override
    public String getInvoke() {
        return "xmasrat";
    }
    @Override
    public Permission requiredPermission() {
        return null;
    }
    @Override
    public Category getCategory() {
        return Category.ANIMALS;
    }
    @Override
    public String getDescription() {
        return "A christmas rat";
    }
}

File: src/main/kotlin/me/nepnep/nepbot/message/command/commands/animals/XmasRat.kt
package me.nepnep.nepbot.message.command.commands.animals

import me.nepnep.nepbot.message.command.Category
import me.nepnep.nepbot.message.command.AbstractCommand
import net.dv8tion.jda.api.entities.channel.middleman.GuildMessageChannel
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class XmasRat : AbstractCommand(
    "xmasrat",
    Category.ANIMALS,
    "A christmas rat"
) {
    override suspend fun execute(args: List<String>, event: MessageReceivedEvent, channel: GuildMessageChannel) {
        channel.sendMessage("https://christmas.bigrat.monster/").queue()
    }
}